2019-06-20 15:12:06 ERROR ConnectionUtil:28 - Could not connect.
2019-06-20 21:20:35 INFO  ConnectionUtil:26 - Connection successful
2019-06-20 22:51:37 INFO  ConnectionUtil:26 - Connection successful
2019-06-21 11:12:29 TRACE UserRepository:28 - Entering create method with parameter: User [firstName=Connor, lastName=Murphy, account=Account [balance=3000000.0, accountID=cm109a2223]]
2019-06-21 11:48:02 TRACE UserRepository:28 - Entering create method with parameter: User [firstName=Connor, lastName=Murphy, account=Account [userName=cmurphy123, password=Password1, accountID=cm109a2223, balance=3000000.0]]
2019-06-21 15:38:55 TRACE UserRepository:28 - Entering create method with parameter: User [firstName=Connor, lastName=Murphy, account=Account [userName=cmurphy123, password=Password1, accountID=cm109a2223, balance=3000000.0]]
2019-06-21 15:38:57 ERROR UserRepository:48 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:43)
	at com.revature.repository.UserRepository.main(UserRepository.java:74)
2019-06-21 15:40:30 TRACE UserRepository:28 - Entering create method with parameter: User [firstName=Connor, lastName=Murphy, account=Account [userName=cmurphy123, password=Password1, accountID=cm109a2223, balance=3000000.0]]
2019-06-24 19:30:42 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-24 19:31:06 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-24 19:45:23 TRACE UserRepository:63 - Entering find user name by name method with paramter: elbio
2019-06-24 19:45:35 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-24 19:45:36 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-24 20:36:36 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-24 20:36:42 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-24 20:36:43 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-24 20:36:50 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-24 20:36:51 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:56)
	at com.revature.controller.Controller.start(Controller.java:55)
	at com.revature.Main.main(Main.java:18)
2019-06-24 21:40:12 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-24 21:40:18 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-24 21:40:18 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-24 21:40:20 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-24 21:40:21 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:56)
	at com.revature.controller.Controller.start(Controller.java:55)
	at com.revature.Main.main(Main.java:18)
2019-06-25 05:15:08 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:15:15 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:15:16 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:15:19 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-25 05:15:20 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:56)
	at com.revature.controller.Controller.start(Controller.java:55)
	at com.revature.Main.main(Main.java:18)
2019-06-25 05:17:17 TRACE UserRepository:63 - Entering find user name by name method with paramter: k
2019-06-25 05:17:19 TRACE UserRepository:90 - Entering find password by name method with username: k
2019-06-25 05:17:20 TRACE UserRepository:90 - Entering find password by name method with username: k
2019-06-25 05:18:26 TRACE UserRepository:63 - Entering find user name by name method with paramter: 3
2019-06-25 05:18:28 TRACE UserRepository:90 - Entering find password by name method with username: 3
2019-06-25 05:18:28 TRACE UserRepository:90 - Entering find password by name method with username: 3
2019-06-25 05:18:32 TRACE UserRepository:117 - Entering update account balance method with new amount -10.0 for username: 3
2019-06-25 05:18:32 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:56)
	at com.revature.controller.Controller.start(Controller.java:55)
	at com.revature.Main.main(Main.java:18)
2019-06-25 05:23:43 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:23:48 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:23:49 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:23:51 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-25 05:23:51 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:56)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 05:25:51 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:25:55 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:25:55 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:27:01 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:27:05 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:33:35 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:33:38 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:33:38 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:35:39 INFO  ConnectionUtil:26 - Connection successful
2019-06-25 05:36:00 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:36:04 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:36:04 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:45:36 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:45:41 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:45:41 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:49:15 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:49:19 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:49:19 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:49:19 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:50:25 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:50:34 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:50:34 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:50:34 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:54:41 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:54:45 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:54:45 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:56:32 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 05:56:35 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:56:35 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:56:35 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 05:56:47 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-25 05:56:47 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:59)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:12:19 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 06:12:23 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:12:23 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:12:24 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:12:27 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-25 06:12:27 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:59)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:12:27 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: rabeet009
2019-06-25 06:12:27 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:60)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:12:27 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password, balance=100.0]]
2019-06-25 06:13:50 TRACE UserRepository:63 - Entering find user name by name method with paramter: rabeet009
2019-06-25 06:24:55 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:25:08 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: rabeet009
2019-06-25 06:25:08 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:25:08 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: rabeet009
2019-06-25 06:25:09 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:25:09 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password, balance=200.0]]
2019-06-25 06:25:09 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:28:29 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:28:30 TRACE UserRepository:90 - Entering find password by name method with username: rabeet009
2019-06-25 06:46:07 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-25 06:46:08 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-25 06:46:08 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-25 06:46:17 TRACE UserRepository:117 - Entering update account balance method with new amount 700.0 for username: elbio
2019-06-25 06:46:17 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:46:17 TRACE UserRepository:117 - Entering update account balance method with new amount 700.0 for username: elbio
2019-06-25 06:46:17 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:46:17 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=elbio, lastName=iseas, account=Account [userName=elbio, password=utah, balance=700.0]]
2019-06-25 06:47:10 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-25 06:47:55 TRACE UserRepository:117 - Entering update account balance method with new amount -150.0 for username: elbio
2019-06-25 06:47:56 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:47:56 TRACE UserRepository:117 - Entering update account balance method with new amount -150.0 for username: elbio
2019-06-25 06:47:56 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 06:47:56 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=elbio, lastName=iseas, account=Account [userName=elbio, password=utah, balance=-150.0]]
2019-06-25 06:47:56 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:19:42 TRACE UserRepository:90 - Entering find password by name method with username: elbio
2019-06-25 07:19:57 TRACE UserRepository:117 - Entering update account balance method with new amount -100.0 for username: elbio
2019-06-25 07:19:58 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:19:58 TRACE UserRepository:117 - Entering update account balance method with new amount -100.0 for username: elbio
2019-06-25 07:19:58 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:19:58 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=elbio, lastName=iseas, account=Account [userName=elbio, password=utah, balance=-100.0]]
2019-06-25 07:19:58 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:24:26 TRACE UserRepository:90 - Entering find password by name method with username: saaim
2019-06-25 07:24:27 TRACE UserRepository:90 - Entering find password by name method with username: saaim
2019-06-25 07:24:28 TRACE UserRepository:90 - Entering find password by name method with username: saaim
2019-06-25 07:24:41 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: saaim
2019-06-25 07:24:42 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:24:42 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: saaim
2019-06-25 07:24:42 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:24:42 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=saaim, lastName=huda, account=Account [userName=saaim, password=password, balance=200.0]]
2019-06-25 07:26:46 TRACE UserRepository:90 - Entering find password by name method with username: saaim
2019-06-25 07:26:47 TRACE UserRepository:90 - Entering find password by name method with username: saaim
2019-06-25 07:53:13 TRACE UserRepository:90 - Entering find password by name method with username: miles
2019-06-25 07:53:15 TRACE UserRepository:90 - Entering find password by name method with username: miles
2019-06-25 07:53:15 TRACE UserRepository:90 - Entering find password by name method with username: miles
2019-06-25 07:53:31 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: miles
2019-06-25 07:53:32 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:53:32 TRACE UserRepository:117 - Entering update account balance method with new amount 200.0 for username: miles
2019-06-25 07:53:33 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:53:33 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=Miles, lastName=Fowler, account=Account [userName=miles, password=passwor2, balance=200.0]]
2019-06-25 07:55:25 TRACE UserRepository:90 - Entering find password by name method with username: miles
2019-06-25 07:55:32 TRACE UserRepository:117 - Entering update account balance method with new amount -100.0 for username: miles
2019-06-25 07:55:33 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:55:33 TRACE UserRepository:117 - Entering update account balance method with new amount -100.0 for username: miles
2019-06-25 07:55:33 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 07:55:33 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=miles, lastName=fowler, account=Account [userName=miles, password=passwor2, balance=-100.0]]
2019-06-25 07:55:33 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:14:26 TRACE UserRepository:90 - Entering find password by name method with username: ty123
2019-06-25 09:14:29 TRACE UserRepository:90 - Entering find password by name method with username: ty123
2019-06-25 09:14:29 TRACE UserRepository:90 - Entering find password by name method with username: ty123
2019-06-25 09:14:48 TRACE UserRepository:117 - Entering update account balance method with new amount 250.0 for username: ty123
2019-06-25 09:14:48 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:14:48 TRACE UserRepository:117 - Entering update account balance method with new amount 250.0 for username: ty123
2019-06-25 09:14:49 ERROR UserRepository:135 - Could not update balance
java.sql.SQLException: ORA-01002: fetch out of sequence

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.fetch(T4CPreparedStatement.java:1022)
	at oracle.jdbc.driver.OracleStatement.fetchMoreRows(OracleStatement.java:3590)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.fetchMoreRows(InsensitiveScrollableResultSet.java:1008)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.absoluteInternal(InsensitiveScrollableResultSet.java:972)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.next(InsensitiveScrollableResultSet.java:572)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:130)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:14:49 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=Tay, lastName=On, account=Account [userName=ty123, password=password2, balance=250.0]]
2019-06-25 09:15:41 TRACE UserRepository:90 - Entering find password by name method with username: ty123
2019-06-25 09:15:59 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: ty123
2019-06-25 09:15:59 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:15:59 TRACE UserRepository:117 - Entering update account balance method with new amount 100.0 for username: ty123
2019-06-25 09:15:59 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:15:59 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=tay, lastName=on, account=Account [userName=ty123, password=password2, balance=100.0]]
2019-06-25 09:16:00 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:21:35 TRACE UserRepository:90 - Entering find password by name method with username: ty123
2019-06-25 09:22:11 TRACE UserRepository:117 - Entering update account balance method with new amount -250.0 for username: ty123
2019-06-25 09:22:11 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:61)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:22:11 TRACE UserRepository:117 - Entering update account balance method with new amount -250.0 for username: ty123
2019-06-25 09:22:11 ERROR UserRepository:135 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:131)
	at com.revature.service.Services.validateBalance(Services.java:62)
	at com.revature.controller.Controller.start(Controller.java:58)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:22:11 TRACE UserRepository:35 - Entering create method with parameter: User [firstName=tay, lastName=on, account=Account [userName=ty123, password=password2, balance=-250.0]]
2019-06-25 09:22:11 ERROR UserRepository:54 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:49)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:44:00 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 09:44:02 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 09:44:12 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 09:44:13 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=Bob, lastName=Fin, account=Account [userName=bfin, password=password3, balance=100.0]]
2019-06-25 09:44:42 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 09:44:50 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 09:44:51 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 09:46:25 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:46:27 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:46:37 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:46:37 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=Ashoka, lastName=Shringla, account=Account [userName=AshokaS, password=password, balance=100.0]]
2019-06-25 09:46:59 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:47:07 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:47:08 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:47:08 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=Ashoka, lastName=Shringla, account=Account [userName=AshokaS, password=password, balance=-50.0]]
2019-06-25 09:47:08 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:56:26 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:56:42 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:56:42 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:56:42 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:56:42 TRACE UserRepository:106 - Entering update account balance method with new amount 200.0 for username: AshokaS
2019-06-25 09:56:42 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:59)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:56:42 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=Ashoka, lastName=Shringla, account=Account [userName=AshokaS, password=password, balance=100.0]]
2019-06-25 09:56:43 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:57:20 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:57:26 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:57:26 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:57:26 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:57:27 TRACE UserRepository:106 - Entering update account balance method with new amount 50.0 for username: AshokaS
2019-06-25 09:57:27 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:59)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:57:27 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=Ashoka, lastName=Shringla, account=Account [userName=AshokaS, password=password, balance=-150.0]]
2019-06-25 09:57:27 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 09:57:58 TRACE UserRepository:79 - Entering find password by name method with username: AshokaS
2019-06-25 09:58:04 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 09:58:04 TRACE UserRepository:132 - Entering get account balance method for username: AshokaS
2019-06-25 14:51:17 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password, balance=-10.0]]
2019-06-25 14:51:19 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 14:54:11 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password, balance=-10.0]]
2019-06-25 14:54:12 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:04:55 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password1, balance=-10.0]]
2019-06-25 15:04:56 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:67)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:07:48 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password1, balance=10.0]]
2019-06-25 15:07:49 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:11:33 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=btut, account=Account [userName=rabeet009, password=password, balance=-10.0]]
2019-06-25 15:11:34 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:66)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:31:07 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=password, balance=0.0]]
2019-06-25 15:31:09 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:60)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:44:48 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=rabeet, lastName=butt, account=Account [userName=rabeet009, password=passwordf, balance=40.0]]
2019-06-25 15:44:49 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:65)
	at com.revature.Main.main(Main.java:18)
2019-06-25 15:56:01 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=joe, lastName=flacco, account=Account [userName=jflac, password=ravens, balance=10000.0]]
2019-06-25 15:56:23 TRACE UserRepository:79 - Entering find password by name method with username: jflac
2019-06-25 15:56:34 TRACE UserRepository:132 - Entering get account balance method for username: jflac
2019-06-25 15:56:34 TRACE UserRepository:132 - Entering get account balance method for username: jflac
2019-06-25 15:58:44 TRACE UserRepository:79 - Entering find password by name method with username: jflac
2019-06-25 15:58:52 TRACE UserRepository:132 - Entering get account balance method for username: jflac
2019-06-25 15:58:52 TRACE UserRepository:132 - Entering get account balance method for username: jflac
2019-06-25 15:58:52 TRACE UserRepository:132 - Entering get account balance method for username: jflac
2019-06-25 15:58:52 TRACE UserRepository:106 - Entering update account balance method with new amount 11000.0 for username: jflac
2019-06-25 15:58:53 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:68)
	at com.revature.controller.Controller.start(Controller.java:84)
	at com.revature.Main.main(Main.java:18)
2019-06-25 16:54:08 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=adrien, lastName=broner, account=Account [userName=ab, password=money, balance=10000.0]]
2019-06-25 16:54:18 TRACE UserRepository:79 - Entering find password by name method with username: adrien
2019-06-25 16:54:18 TRACE UserRepository:79 - Entering find password by name method with username: adrien
2019-06-25 17:13:09 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=peter, lastName=alagna, account=Account [userName=peter, password=trainer, balance=1000.0]]
2019-06-25 17:13:24 TRACE UserRepository:79 - Entering find password by name method with username: peter
2019-06-25 17:13:24 TRACE UserRepository:79 - Entering find password by name method with username: peter
2019-06-25 17:14:09 TRACE UserRepository:79 - Entering find password by name method with username: rabeet009
2019-06-25 17:14:20 TRACE UserRepository:132 - Entering get account balance method for username: rabeet009
2019-06-25 17:14:20 TRACE UserRepository:132 - Entering get account balance method for username: rabeet009
2019-06-25 17:14:20 TRACE UserRepository:132 - Entering get account balance method for username: rabeet009
2019-06-25 17:14:20 TRACE UserRepository:106 - Entering update account balance method with new amount 1500.0 for username: rabeet009
2019-06-25 17:14:21 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:83)
	at com.revature.Main.main(Main.java:18)
2019-06-25 18:53:20 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=test, lastName=case, account=Account [userName=tcase, password=t1, balance=50.0]]
2019-06-25 18:53:32 TRACE UserRepository:79 - Entering find password by name method with username: tcase
2019-06-25 18:54:00 TRACE UserRepository:132 - Entering get account balance method for username: tcase
2019-06-25 18:54:00 TRACE UserRepository:132 - Entering get account balance method for username: tcase
2019-06-25 18:54:02 TRACE UserRepository:132 - Entering get account balance method for username: tcase
2019-06-25 18:54:03 TRACE UserRepository:106 - Entering update account balance method with new amount 150.0 for username: tcase
2019-06-25 18:54:07 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:83)
	at com.revature.Main.main(Main.java:18)
2019-06-25 18:54:07 TRACE UserRepository:132 - Entering get account balance method for username: tcase
2019-06-25 19:31:22 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 19:35:02 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 19:35:04 TRACE UserRepository:79 - Entering find password by name method with username: bfin
2019-06-25 20:01:06 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:01:09 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:01:09 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:01:09 TRACE UserRepository:106 - Entering update account balance method with new amount 150.0 for username: bfin
2019-06-25 20:01:10 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.eval.project0.EvaluationServiceTest.successfulUpdate(EvaluationServiceTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-25 20:02:01 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:02:03 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:05:53 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:05:54 TRACE UserRepository:132 - Entering get account balance method for username: bfin
2019-06-25 20:06:29 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=3, lastName=3, account=Account [userName=3, password=3, balance=3.0]]
2019-06-25 20:14:11 TRACE UserRepository:52 - Entering find user name by name method with paramter: rabeet009
2019-06-25 20:14:53 TRACE UserRepository:52 - Entering find user name by name method with paramter: tcase2
2019-06-25 20:15:13 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=test2, lastName=case2, account=Account [userName=tcase2, password=done, balance=100000.0]]
2019-06-25 20:15:28 TRACE UserRepository:79 - Entering find password by name method with username: tcase2
2019-06-25 20:15:34 TRACE UserRepository:132 - Entering get account balance method for username: tcase2
2019-06-25 20:15:35 TRACE UserRepository:132 - Entering get account balance method for username: tcase2
2019-06-25 20:15:35 TRACE UserRepository:132 - Entering get account balance method for username: tcase2
2019-06-25 20:15:35 TRACE UserRepository:106 - Entering update account balance method with new amount 99497.0 for username: tcase2
2019-06-25 20:15:35 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:86)
	at com.revature.Main.main(Main.java:18)
2019-06-25 20:15:35 TRACE UserRepository:132 - Entering get account balance method for username: tcase2
2019-06-25 22:04:14 TRACE UserRepository:52 - Entering find user name by name method with paramter: abee
2019-06-25 22:04:37 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=apple, lastName=bee, account=Account [userName=abee, password=yummy, balance=100.0]]
2019-06-25 22:05:01 TRACE UserRepository:79 - Entering find password by name method with username: apple
2019-06-25 22:05:01 TRACE UserRepository:79 - Entering find password by name method with username: apple
2019-06-25 22:05:18 TRACE UserRepository:132 - Entering get account balance method for username: apple
2019-06-25 22:05:19 TRACE UserRepository:132 - Entering get account balance method for username: apple
2019-06-25 22:23:24 TRACE UserRepository:52 - Entering find user name by name method with paramter: abee
2019-06-25 22:25:57 TRACE UserRepository:52 - Entering find user name by name method with paramter: fake1
2019-06-25 22:25:58 TRACE UserRepository:52 - Entering find user name by name method with paramter: fake1
2019-06-25 22:30:31 TRACE UserRepository:52 - Entering find user name by name method with paramter: fake1
2019-06-25 22:30:32 TRACE UserRepository:52 - Entering find user name by name method with paramter: fake1
2019-06-25 22:31:14 TRACE UserRepository:52 - Entering find user name by name method with paramter: jdoe1
2019-06-25 22:31:17 TRACE UserRepository:52 - Entering find user name by name method with paramter: jdoe1
2019-06-25 22:31:32 TRACE UserRepository:52 - Entering find user name by name method with paramter: jdoe
2019-06-25 22:31:34 TRACE UserRepository:52 - Entering find user name by name method with paramter: jdoe
2019-06-25 22:36:20 TRACE UserRepository:52 - Entering find user name by name method with paramter: jbrabo
2019-06-25 22:36:22 TRACE UserRepository:52 - Entering find user name by name method with paramter: jbrabo
2019-06-25 22:37:22 TRACE UserRepository:52 - Entering find user name by name method with paramter: jdoe1
2019-06-25 22:38:43 TRACE UserRepository:52 - Entering find user name by name method with paramter: jbravo
2019-06-25 22:38:45 TRACE UserRepository:52 - Entering find user name by name method with paramter: jbravo
2019-06-25 22:58:14 TRACE UserRepository:52 - Entering find user name by name method with paramter: pstar
2019-06-25 22:58:30 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=patrick, lastName=statr, account=Account [userName=pstar, password=pink, balance=100000.0]]
2019-06-25 22:58:40 TRACE UserRepository:79 - Entering find password by name method with username: pstar231
2019-06-25 22:58:40 TRACE UserRepository:79 - Entering find password by name method with username: pstar231
2019-06-25 22:58:46 TRACE UserRepository:132 - Entering get account balance method for username: pstar231
2019-06-25 22:58:46 TRACE UserRepository:132 - Entering get account balance method for username: pstar231
2019-06-25 23:21:39 TRACE UserRepository:52 - Entering find user name by name method with paramter: final1
2019-06-25 23:21:40 TRACE UserRepository:79 - Entering find password by name method with username: final1
2019-06-25 23:25:02 TRACE UserRepository:132 - Entering get account balance method for username: fina1
2019-06-25 23:25:03 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=final, lastName=test, account=Account [userName=fina1, password=done, balance=1000.0]]
2019-06-25 23:25:24 TRACE UserRepository:52 - Entering find user name by name method with paramter: finally
2019-06-25 23:25:24 TRACE UserRepository:79 - Entering find password by name method with username: finally
2019-06-26 00:29:06 TRACE UserRepository:132 - Entering get account balance method for username: n.uzu
2019-06-26 00:29:07 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=naruto, lastName=uzumaki, account=Account [userName=n.uzu, password=sasuke, balance=1000.0]]
2019-06-26 00:29:18 TRACE UserRepository:52 - Entering find user name by name method with paramter: n.uzu
2019-06-26 00:35:09 TRACE UserRepository:52 - Entering find user name by name method with paramter: mike1
2019-06-26 00:35:10 TRACE UserRepository:79 - Entering find password by name method with username: mike1
2019-06-26 00:43:55 TRACE UserRepository:79 - Entering find password by name method with username: n.uzu
2019-06-26 00:44:36 TRACE UserRepository:132 - Entering get account balance method for username: n.uzu
2019-06-26 00:44:37 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=naruto, lastName=uzumaki, account=Account [userName=n.uzu, password=sasuke, balance=10000.0]]
2019-06-26 00:44:37 ERROR UserRepository:43 - Could not create user account.
java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (BANK_DB.PK_USER) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.repository.UserRepository.create(UserRepository.java:38)
	at com.revature.controller.Controller.start(Controller.java:69)
	at com.revature.Main.main(Main.java:18)
2019-06-26 00:59:34 TRACE UserRepository:52 - Entering find user name by name method with paramter: r.coleman
2019-06-26 00:59:47 TRACE UserRepository:132 - Entering get account balance method for username: r.coleman
2019-06-26 00:59:48 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=ronny, lastName=coleman, account=Account [userName=r.coleman, password=champ, balance=10000.0]]
2019-06-26 00:59:59 TRACE UserRepository:79 - Entering find password by name method with username: r.coleman
2019-06-26 01:00:13 TRACE UserRepository:132 - Entering get account balance method for username: r.coleman
2019-06-26 01:00:13 TRACE UserRepository:132 - Entering get account balance method for username: r.coleman
2019-06-26 01:00:14 TRACE UserRepository:132 - Entering get account balance method for username: r.coleman
2019-06-26 01:00:14 TRACE UserRepository:106 - Entering update account balance method with new amount 5000.0 for username: r.coleman
2019-06-26 01:00:15 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:69)
	at com.revature.controller.Controller.start(Controller.java:88)
	at com.revature.Main.main(Main.java:18)
2019-06-26 01:00:15 TRACE UserRepository:132 - Entering get account balance method for username: r.coleman
2019-06-26 01:07:13 TRACE UserRepository:52 - Entering find user name by name method with paramter: nuser
2019-06-26 01:07:27 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=new, lastName=user, account=Account [userName=nuser, password=new1233, balance=1000.0]]
2019-06-26 01:07:28 TRACE UserRepository:132 - Entering get account balance method for username: nuser
2019-06-26 01:07:39 TRACE UserRepository:79 - Entering find password by name method with username: user
2019-06-26 01:07:39 TRACE UserRepository:79 - Entering find password by name method with username: user
2019-06-26 01:11:48 TRACE UserRepository:52 - Entering find user name by name method with paramter: j.seed
2019-06-26 01:12:00 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=johnny, lastName=appleseed, account=Account [userName=j.seed, password=apples, balance=500.0]]
2019-06-26 01:12:00 TRACE UserRepository:132 - Entering get account balance method for username: j.seed
2019-06-26 01:12:07 TRACE UserRepository:79 - Entering find password by name method with username: j.seed
2019-06-26 01:22:16 TRACE UserRepository:79 - Entering find password by name method with username: rabeet009
2019-06-26 01:22:21 TRACE UserRepository:132 - Entering get account balance method for username: rabeet009
2019-06-26 01:22:22 TRACE UserRepository:132 - Entering get account balance method for username: rabeet009
2019-06-26 07:33:52 INFO  ConnectionUtil:26 - Connection successful
2019-06-26 07:33:52 INFO  ConnectionUtil:26 - Connection successful
2019-06-26 07:34:06 INFO  ConnectionUtil:26 - Connection successful
2019-06-26 07:34:15 INFO  ConnectionUtil:26 - Connection successful
2019-06-26 07:34:50 TRACE UserRepository:52 - Entering find user name by name method with paramter: mtest
2019-06-26 07:35:05 TRACE UserRepository:24 - Entering create method with parameter: User [firstName=morning, lastName=test, account=Account [userName=mtest, password=pass1, balance=1000.0]]
2019-06-26 07:35:05 TRACE UserRepository:132 - Entering get account balance method for username: mtest
2019-06-26 07:35:15 TRACE UserRepository:79 - Entering find password by name method with username: mtest
2019-06-26 07:35:21 TRACE UserRepository:132 - Entering get account balance method for username: mtest
2019-06-26 07:35:22 TRACE UserRepository:132 - Entering get account balance method for username: mtest
2019-06-26 07:35:22 TRACE UserRepository:132 - Entering get account balance method for username: mtest
2019-06-26 07:35:22 TRACE UserRepository:106 - Entering update account balance method with new amount 500.0 for username: mtest
2019-06-26 07:35:22 ERROR UserRepository:124 - Could not update balance
java.sql.SQLSyntaxErrorException: ORA-00900: invalid SQL statement

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Odscrarr.doODNY(T4C8Odscrarr.java:96)
	at oracle.jdbc.driver.T4CPreparedStatement.doDescribe(T4CPreparedStatement.java:719)
	at oracle.jdbc.driver.OracleStatement.getColumnIndex(OracleStatement.java:3951)
	at oracle.jdbc.driver.InsensitiveScrollableResultSet.findColumn(InsensitiveScrollableResultSet.java:299)
	at oracle.jdbc.driver.GeneratedResultSet.getDouble(GeneratedResultSet.java:1328)
	at com.revature.repository.UserRepository.updateBalance(UserRepository.java:120)
	at com.revature.service.Services.validateBalance(Services.java:66)
	at com.revature.controller.Controller.start(Controller.java:91)
	at com.revature.Main.main(Main.java:18)
2019-06-26 07:35:22 TRACE UserRepository:132 - Entering get account balance method for username: mtest
